### DISCUSSION TITLE
How to temporarily disable notification?

### DISCUSSION THREAD

**Discussion by Mobilizes** [LazyVim/LazyVim#5229] [Q&A] [answered]
Hello, I am pretty clueless on this issue
```lua
local Snacks = require("snacks")
local copilot_exists = pcall(require, "copilot")

if copilot_exists then
  ...
  require("copilot.command").disable()
end
```

I like my copilot to be disabled by default, but every start up, there is an annoying notification that blocks my neo-tree that I set to be on the right.
![image](https://github.com/user-attachments/assets/01924000-2839-4149-bfef-d078976c24d2)

How do I temporarily disable this notification?

**Comment by dpetka2001** [comment]
https://github.com/folke/snacks.nvim/blob/main/docs/toggle.md#%EF%B8%8F-config 

It seems it also takes a `notify` field. Should do what you want?

**Reply by Mobilizes** [reply]
Sorry, I was not clear enough with my previous question. I wanted to disable the notification caused by this line.
```lua
require("copilot.command").disable()
```

**Reply by dpetka2001** [reply, marked as answer]
That notification comes straight from Copilot itself (see [here](https://github.com/zbirenbaum/copilot.lua/blob/d3783b9283a7c35940ed8d71549030d5f5f9f980/lua/copilot/command.lua#L87-L90)). Nothing to do with Snacks or LazyVim. You could try something like the following
```lua
return {
	{
		"folke/noice.nvim",
		opts_extend = { "routes" },
		opts = {
			routes = {
				{
					filter = {
						event = "msg_show",
						any = {
							{ find = "Offline" },
						},
					},
					opts = { skip = true },
				},
			},
		},
	},
}
```
But then that notification would never show up even when you did `:Copilot status`. Not sure that's what you want. If you want some other behavior you would better ask at `copilot.lua` github repo, so that maybe the Notifications it shows could be configurable by the user.

**Reply by Mobilizes** [reply]
Yeah, I think this will do for now, thank you!
